nf_wochenende.nf

```diff
@@ -124,28 +124,66 @@ workflow {
     println "########### End settings ##############"
 
     // Parameters - throw warnings at present
+    // Need to remap the boolean parameters and create a new py parameter that python can understand. eg longread = true: pylongread = "--longread", false: pylongread = ""
+    longread = true             // Are reads from ONT or Pacbio? Recommend minimap2long aligner
+    no_dup_removal = false      // Do not remove duplicate reads
+    nextera = false             // Use illumina nextera adapter trimming
+    abra = false                // Use abra realignment (True, False)
+    no_prinseq = true           // Filter reads using prinseq (only for short reads)
+    no_fastqc = true            // Do not run fastqc
+    fastp = false               // Use fastp trimming tool (short reads)
+    trim_galore = false         // Use trim_galore trimmer (short reads)
+    //  usage: run_Wochenende.py [-h] [--aligner {bwamem,minimap2short,minimap2long,ngmlr}] [--readType {PE,SE}] [--ref REF] [--threads THREADS] [--fastp] [--nextera] [--trim_galore] [--debug] [--longread]
+    //                       [--no_duplicate_removal] [--no_prinseq] [--no_fastqc] [--no_abra] [--mq20] [--mq30] [--remove_mismatching REMOVE_MISMATCHING] [--force_restart]
+    //                       fastq
+
+
     if (params.mapping_quality != "") {
-       params.mq = "--" + params.mapping_quality
+       params.py_mq = "--" + params.mapping_quality
     } else {
-       params.mq = ""
+       params.py_mq = ""
     }
 
     if (params.no_abra) {
```

Comment:
I think `no_abra` has to be changed to `abra`

Reply:
true, thanks, removed some working comments from the nextflow.config too. This is not tested yet.